src/languages/es.ts

```diff
@@ -3693,6 +3693,35 @@ export default {
                 title: 'Informes de gastos',
                 subtitle: 'Automatiza el cumplimiento, la aprobación y el pago de los informes de gastos.',
             },
+            categoryRules: {
```

Comment:
are those from the gpt, or confirmed?

Reply:
@pecanoro Could you take a look at these translations?